doc/indent-example.c

```diff
@@ -0,0 +1,45 @@
+#include "r_core.h"
+#include "r_util.h"
+#include <stdio.h>
+
+typedef struct r_core_rtr_host_t2 {
+	int proto;
+	char host[512];
+	int port;
+	char file[1024];
+	RSocket *fd;
+} RCoreRtrHost2;
+
+static const char *help_msg_aa[] = {
+	"Usage:", "aa[0*?]", " # see also 'af' and 'afna'",
+	"aa", " ", "alias for 'af@@ sym.*;af@entry0;afva'", //;.afna @@ fcn.*'",
+	"aa*", "", "analyze all flags starting with sym. (af @@ sym.*)",
+	NULL};
```

Comment:
the } should be in the next line

Reply:
} shuld be in the next line

Reply:
As said, is this really important? What I want is *one* way to automatically indent/fix style in my code. I don't care whether there's a space before/after a given word, as long as there's a tool that automagically does that.

Is there such a tool?
If yes, perfect, please show me.
If not, I think it's time we change a bit our coding style to adapt to those existing tools.

Reply:
well it looks bad to me, and i dont know if there's any tool to do this unless my sys/indent.sh was doing it, which i didnt checked. i dont think we should change our coding style because we need to make code beautiful to read and not adjust to the bugs of the indentation tools. again, i didnt had time to test this pr yet, just speaking out loud

Reply:
The `}` is probably attached because there is no `,` after NULL. Maybe we could programmatically insert colons at the end of lists like this to force the `}` to go to the next line.

On the other hand, I agree with retlibc that the attached-closing-curly-bracket-style is also OK, you would just need to get used to it.